runtime/gc_modron_startup/mminit.cpp

```diff
@@ -3200,9 +3200,9 @@ gcReinitializeDefaultsForRestore(J9VMThread* vmThread)
 	 */
 	extensions->usablePhysicalMemory = omrsysinfo_get_addressable_physical_memory();
 	/* If the thread count is being forced, check its validity and display a warning message if it is invalid, then mark it as invalid. */
-	if (extensions->gcThreadCountForced && (extensions->gcThreadCount < extensions->dispatcher->threadCountMaximum())) {
+	if (extensions->gcThreadCountSpecified && (extensions->gcThreadCount < extensions->dispatcher->threadCountMaximum())) {
 		j9nls_printf(PORTLIB, J9NLS_WARNING, J9NLS_GC_THREAD_VALUE_MUST_BE_ABOVE_WARN, (UDATA)extensions->dispatcher->threadCountMaximum());
-		extensions->gcThreadCountForced = false;
+		extensions->gcThreadCountSpecified = false;
```

Comment:
not sure if this was correct originally, but we don't want to reset this to false, anymore

in general 'specified' attributes, should i be just set during parsing, and left unchanged afterwards